src/host/polyfills/simpleView.ts

```diff
@@ -114,6 +114,15 @@ export function applyStylesToggleFocusPatch(content: string): string | null {
     return replaceInSourceCode(content, pattern, replacementText);
 }
 
+export function applyNoMatchingStylesPatch(content: string): string | null {
+    // Patch to inform user to refresh/resume target to get CSS information when attaching to a paused target.
+    const pattern = /this\._noMatchesElement\.textContent\s*=\s*(ls\s*`No matching selector or style`);/g;
+    const replacementText = `
+       this._noMatchesElement.innerHTML = ls \`No matching selector or style.<br />Styles may not be available if target was paused when opening Edge DevTools.<br />Please resume or refresh the target.\`;
```

Comment:
layout should be done outside of localization messages as the library does not support it. So in this case this needs to be split in 3 different parts.

Reply:
Ahh, good call. Wasn't sure the best way to do this, but that makes alot of sense. I'll go ahead and update